src/Foundation/NSAttributedString.mac.cs

```diff
@@ -296,6 +296,14 @@ public partial class NSAttributedStringDocumentAttributes : DictionaryContainer
 			get { return GetFloatValue (NSStringAttributeKey.NSTimeoutDocumentOption); }
 			set { SetNumberValue (NSStringAttributeKey.NSTimeoutDocumentOption, (float?)value); }
 		}
+
+		public NSTextScalingType SourceTextScaling {
+			get { return (NSTextScalingType)GetLongValue (NSStringAttributeKey.NSSourceTextScalingDocumentOption); }
+		}
+
+		public NSTextScalingType TargetTextScaling {
+			get { return (NSTextScalingType)GetLongValue (NSStringAttributeKey.NSTargetTextScalingDocumentOption); }
+		}
```

Comment:
The header says - 
```
APPKIT_EXTERN NSAttributedStringDocumentReadingOptionKey const NSTargetTextScalingDocumentOption API_AVAILABLE(macos(10.15), ios(13.0));  // @"TargetTextScaling", one of the text scaling types declared above.  Only affects RTF documents.  For reading methods, you can pass this option to request that the returned attributed string uses the specified scaling.  The font point sizes in the document may be converted if necessary.  If this option is not provided, the system will deduce the target text scaling type based on application framework and platform.
```
which i believe refers to the `NSTextScalingType` enum defined here.

Reply:
are you sure they are ready-only ? that does not seem very useful

Reply:
Its a `const` actually